test/e2e/generated/gobindata_util.go

```diff
@@ -14,24 +14,23 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package framework
+package generated
 
-//go:generate ../../../hack/update-bindata.sh
+import "k8s.io/kubernetes/test/e2e/framework"
```

Comment:
Yeah that makes sense for bindata_util . Framework logging functions aren't that useful anyway :)